custom-words.txt

```diff
@@ -2481,6 +2481,7 @@ Wmem
 wordprocessingml
 workbooktemplates
 Workernode
+workflowsconfiguration
```

Comment:
Don't add this term, use proper camelCase, PascalCase to enable term recognition, or in descriptions, use normal English "workflow's configuration" etc.

Reply:
Good call! Fixed